spec/test/vue_js/onclick/onclick_spec.rb

```diff
@@ -1,46 +1,148 @@
 require 'rails_vue_js_spec_helper'
 include VueJsSpecUtils
 
-describe "Onclick Component", type: :feature, js: true do
+describe "Onclick Component", type: :feature, js: true, aggregate_failures: true do
```

Comment:
@jonasjabari Given these are Acceptance specs and we're trying to perform multiple expectations on the the same example (it makes sense to do so in my opinion, as is the convention in other specs in the project), I added this tag to the example group.

This would most likely do well with the other example groups throughout the spec suite as we have multiple expectations per example and it'd be nice to know all/which of those expectations are the ones failing (if they do). Just a quality-of-life improvement in my opinion.